workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowLifecycleOwner.kt

```diff
@@ -229,18 +247,20 @@ internal class RealWorkflowLifecycleOwner(
         parentLifecycle?.removeObserver(this)
         parentLifecycle = null
 
-        // We can't change state anymore, so we don't care about watching for new parents.
-        view.get()?.removeOnAttachStateChangeListener(this)
```

Comment:
I think we can restore this, can't hurt to best-effort try to remove this listener.